tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="cp319" name="Cellprofiler" version="19.05">
+  <description>Run a CellProfiler v3.1.9 pipeline</description>
+  <requirements>
+     <requirement type="package" version="3.1.9">cellprofiler</requirement>
+  </requirements>
+  <configfiles>
+<configfile name="script_file">
+mkdir `pwd`/input
+mkdir `pwd`/output
+#for $i,$filename in enumerate($inputType.inputdir):
+ln -s $filename `pwd`/input/${filename.element_identifier}.${filename.ext}
+#end for
+
+find `pwd`/input -name *.tiff > list.txt
+
+</configfile>
+  </configfiles>
+
+  <command><![CDATA[
+      bash '$script_file' &&
+      cellprofiler -c -r --file-list list.txt -o `pwd`/output -p $pipeline 
```

Comment:
* remove pwd here
* $pipeline sould be single-quoted

Reply:
updated